lib/src/maintenance.dart

```diff
@@ -100,8 +100,8 @@ class Maintenance extends Object with _$MaintenanceSerializerMixin {
   factory Maintenance.fromJson(Map<String, dynamic> json) =>
       _$MaintenanceFromJson(json);
 
-  double getMaintenanceScore(DateTime publishDate) {
-    final now = new DateTime.now().toUtc();
+  double getMaintenanceScore(DateTime publishDate, {DateTime now}) {
```

Comment:
Would be to have `Duration age` as the only parameter?

Reply:
Ha! that's next

Reply:
Wanted to avoid breaks until we got good tests...

LGTY?

Reply:
please add a TODO there